packages/minimongo/minimongo.js

```diff
@@ -957,6 +965,14 @@ LocalCollection._updateInResults = function (query, doc, old_doc) {
 // oldResults is guaranteed to be ignored if the query is not paused.
 LocalCollection.prototype._recomputeResults = function (query, oldResults) {
   var self = this;
+  if (self.paused) {
+    // There's no reason to recompute the results now as we're still paused.
+    // By flagging the query as "dirty", the recompute will be performed
+    // when resumeObservers is called.
+    query.dirty = true;
```

Comment:
It might be worth clearing `query.results` here?

```js
      if (query.ordered) {
        query.results = [];
      } else {
        query.results.clear();
      }
```